presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcMetadata.java

```diff
@@ -176,15 +176,38 @@ public void getColumnMetadata()
                 new ColumnMetadata("text", VARCHAR));
     }
 
-    @Test(expectedExceptions = PrestoException.class)
-    public void testCreateTable()
+    @Test
+    public void testCreateAndAlterTable()
     {
-        metadata.createTable(
-                SESSION,
-                new ConnectorTableMetadata(
-                        new SchemaTableName("example", "foo"),
-                        ImmutableList.of(new ColumnMetadata("text", VARCHAR))),
-                false);
+        SchemaTableName table = new SchemaTableName("example", "foo");
+        metadata.createTable(SESSION, new ConnectorTableMetadata(table, ImmutableList.of(new ColumnMetadata("text", VARCHAR))), false);
```

Comment:
`text` is a confusing column name since it is a data type in PostgreSQL

Reply:
It was text before my changes, may follow up on this separately.